### DISCUSSION TITLE
Does ioter support silabs thunderboard sense 2?

### DISCUSSION THREAD

**Discussion by ejunu** [Samsung/ioter#79] [Q&A] [answered]
@junyong-sim 
Silabs thunderboard sense 2 also seems to support thread rcp. Please check.

[Silabs thunderboard sense 2](https://www.silabs.com/development-tools/wireless/efr32xg24-dev-kit?tab=overview)

**Comment by junyong-sim** [comment, marked as answer]
Yeah, when I've tested ioter with thunderboard sense 2, it's working well.
But, it's a litter slower than nordic or qorvo.
